DATA/crustmap/readme.txt

```diff
@@ -28,7 +28,7 @@ density    crustmaps (e.g. eucrustr3.cmap - eucrustr7.cmap)
 
 these are simple ascii data files and you can change them easily. The current crustmap resolution
 is set to 1/4 degree.
-The Values are ordered on a grid that runs from 90° lat (0° lng to 180° lng ) to -90° lat.
+The Values are ordered on a grid that runs from 90? lat (0? lng to 180? lng ) to -90? lat.
```

Comment:
Thanks! Fixed as well.
The problem came from the new non-ASCII characters cleaning routine, 
which does not handle the "°" character for "degrees".


On 01/22/2015 04:49 AM, Elliott Sales de Andrade wrote:
> In DATA/crustmap/readme.txt
> <https://github.com/geodynamics/specfem3d_globe/pull/308#discussion_r23354128>:
>
>> @@ -28,7 +28,7 @@ density    crustmaps (e.g. eucrustr3.cmap - eucrustr7.cmap)
>>
>>  these are simple ascii data files and you can change them easily. The current crustmap resolution
>>  is set to 1/4 degree.
>> -The Values are ordered on a grid that runs from 90° lat (0° lng to 180° lng ) to -90° lat.
>> +The Values are ordered on a grid that runs from 90? lat (0? lng to 180? lng ) to -90? lat.
>
> Should write out degrees here, or just remove the |?| since it looks
> like you aren't sure whether it's 90.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem3d_globe/pull/308/files#r23354128>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr